server/spinwick.go

```diff
@@ -145,22 +145,22 @@ func (s *Server) createSpinWick(pr *model.PullRequest, size string, withLicense
 	}
 	cloudClient := cloudModel.NewClientWithHeaders(s.Config.ProvisionerServer, headers)
 
-	var groupID string
-	var group *cloudModel.Group
-	if len(s.Config.CloudGroupID) != 0 {
-		group, err = cloudClient.GetGroup(s.Config.CloudGroupID)
-		if err != nil {
-			return request.WithError(errors.Wrapf(err, "unable to get group with ID %s", s.Config.CloudGroupID))
-		}
-		if group == nil {
-			return request.WithError(fmt.Errorf("group with ID %s does not exist", s.Config.CloudGroupID))
-		}
-		groupID = s.Config.CloudGroupID
-	}
+	// TODO: (cpanato) add the group permission in the AUTH
+	// var groupID string
+	// var group *cloudModel.Group
+	// if len(s.Config.CloudGroupID) != 0 {
+	// 	group, err = cloudClient.GetGroup(s.Config.CloudGroupID)
+	// 	if err != nil {
+	// 		return request.WithError(errors.Wrapf(err, "unable to get group with ID %s", s.Config.CloudGroupID))
+	// 	}
+	// 	if group == nil {
+	// 		return request.WithError(fmt.Errorf("group with ID %s does not exist", s.Config.CloudGroupID))
+	// 	}
+	// 	groupID = s.Config.CloudGroupID
+	// }
 
 	installationRequest := &cloudModel.CreateInstallationRequest{
 		OwnerID:  ownerID,
-		GroupID:  groupID,
```

Comment:
You removed the line that set the groupID, but we still need to set it here. :)

Reply:
added that below line 175 @gabrieljackson

Reply:
Oh! Yeah, I see that now. Thanks for pointing me to it.